posthog/clickhouse/plugin_log_entries.py

```diff
@@ -3,11 +3,13 @@
 from posthog.kafka_client.topics import KAFKA_PLUGIN_LOG_ENTRIES
 from posthog.settings import CLICKHOUSE_CLUSTER, CLICKHOUSE_DATABASE
 
+ON_CLUSTER_CLAUSE = lambda: f"ON CLUSTER '{CLICKHOUSE_CLUSTER}'"
```

Comment:
```suggestion
ON_CLUSTER_CLAUSE = lambda on_cluster=True: f"ON CLUSTER '{CLICKHOUSE_CLUSTER}'" if on_cluster else ""
```

It would make checking the condition in just 1 place instead of all the below SQL queries. Less chance of a mistake.

Reply:
done!

I need the lambdas though, because the statements are the ones that are called with the on_cluster clause or not in the migrations.